cloudinit/sources/__init__.py

```diff
@@ -925,6 +925,10 @@ def check_instance_id(self, sys_cfg):
         # quickly (local check only) if self.instance_id is still
         return False
 
+    def check_fallback(self):
```

Comment:
Could we make this method name more descriptive please? Also we should add a docstring that will communicate to other datasource authors what the purpose of this method is. There are many fallbacks in cloud-init, so this isn't very meaningful on its own.

Reply:
Thanks for the review.
Yes, I've renamed this method(please suggest a better name if needed), and added a docstring.